### PR TITLE
Bitcoin Core 24.1

### PR DESCRIPTION
**Pull request by fanquake**
Add release pages for Bitcoin Core `v24.1`.

### CODE REVIEW

**Comment by katesalazar** [comment]
Congrats on the new release!

File: _releases/24.1.md [hunk 1]
```diff
@@ -0,0 +1,127 @@
+---
+title: Bitcoin Core 24.1
+id: en-release-24.1
+name: release-24.1
+permalink: /en/releases/24.1/
+excerpt: Bitcoin Core version 24.1 is now available
+date: 2023-05-16
```
Comment (glozow) [inline _releases/24.1.md:7]: nit: 2023-05-18?

Comment (fanquake) [reply _releases/24.1.md]: Fixed.

**Review by glozow — COMMENTED** [review]
ACK e9e0e69dc6b715aa8c58b92b824b5a74e485da52

**Comment by hebasto** [comment]
Why do we point Bitcoin Core users to #bitcoin IRC chat?

File: _releases/24.1.md [hunk 2]
```diff
@@ -0,0 +1,127 @@
...
+## Use the same number of elements as decimal places, e.g. "0.1.2 => [0,
+## 1, 2]" versus "1.2 => [1, 2]"
+release: [24, 1]
+
+## Optional magnet link.  To get it, open the torrent in a good BitTorrent client
+## and View Details, or install the transmission-cli Debian/Ubuntu package
+## and run: transmission-show -m <torrent file>
+#
+## Link should be enclosed in quotes and start with: "magnet:?
+optional_magnetlink: "magnet:?xt=urn:btih:ebb58d7495a8aaed2f20ec4ce3e5ae27aff69529&dn=bitcoin-core-24.1&tr=udp%3A%2F%2Ftracker.openbittorrent.com%3A80&tr=udp%3A%2F%2Ftracker.opentrackr.org%3A1337%2Fannounce&tr=udp%3A%2F%2Ftracker.coppersurfer.tk%3A6969%2Fannounce&tr=udp%3A%2F%2Ftracker.leechers-paradise.org%3A6969%2Fannounce&tr=udp%3A%2F%2Fexplodie.org%3A6969%2Fannounce&tr=udp%3A%2F%2Ftracker.torrent.eu.org%3A451%2Fannounce&tr=udp%3A%2F%2Ftracker.bitcoin.sprovoost.nl%3A6969"
```
Comment (hebasto) [inline _releases/24.1.md:20]: FWIW, I'm not able to connect to udp://tracker.coppersurfer.tk:6969 and udp://tracker.leechers-paradise.org:6969 trackers.

**Review by hebasto — APPROVED** [review]
ACK e9e0e69dc6b715aa8c58b92b824b5a74e485da52, I've downloaded binaries via the torrent magnetic link and verified their hashes.

**Pull request merged**